common/djangoapps/student/helpers.py

```diff
@@ -98,6 +105,10 @@ def check_verify_status_by_course(user, course_enrollments):
 
             relevant_verification = SoftwareSecurePhotoVerification.verification_for_datetime(deadline, verifications)
 
+            verification_expiring_soon = False
+            if SoftwareSecurePhotoVerification.is_verification_expiring_soon(expiration_datetime):
```

Comment:
@tasawernawaz why can't this be `verification_expiring_soon = SoftwareSecurePhotoVerification.is_verification_expiring_soon(expiration_datetime)`